Missing sctp.h not detected by cmakefiles

```
$ ./build.sh --full
-- Checking for one of the modules 'libpcap'
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
GSLCBLAS_LIBRARY
    linked by target "sipp_unittest" in directory ~src/sipp
    linked by target "sipp" in directory ~/src/sipp
GSL_LIBRARY
    linked by target "sipp_unittest" in directory ~/src/sipp
    linked by target "sipp" in directory ~/src/sipp

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```
```
$ sudo apt-get install libgsl-dev
...
```
```
$ ./build.sh --full
-- Configuring done
...
In file included from ~/src/sipp/src/actions.cpp:24:
~/src/sipp/include/sipp.hpp:36:10: fatal error: netinet/sctp.h: No such file or directory
   36 | #include <netinet/sctp.h>
      |          ^~~~~~~~~~~~~~~~
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -302,10 +302,8 @@ endif(USE_PCAP)
 
 if(USE_SCTP)
   find_library(SCTP_LIBRARY sctp)
-  if(SCTP_LIBRARY)
-    target_link_libraries(sipp ${SCTP_LIBRARY})
-    target_link_libraries(sipp_unittest ${SCTP_LIBRARY})
-  endif()
+  target_link_libraries(sipp ${SCTP_LIBRARY})
+  target_link_libraries(sipp_unittest ${SCTP_LIBRARY})
 endif()
 
 if(BUILD_STATIC AND MSYS)

